apple/RNGestureHandlerButtonComponentView.mm

```diff
@@ -21,6 +21,11 @@ @implementation RNGestureHandlerButtonComponentView {
   RNGestureHandlerButton *_buttonView;
 }
 
++ (BOOL)shouldBeRecycled
+{
+  return NO;
+}
```

Comment:
Can we do this only on macOS?

Reply:
Sure, done in [29a3356](https://github.com/software-mansion/react-native-gesture-handler/pull/3154/commits/29a3356496840ce6d8eacc5c384ad32df35795c2)

Reply:
Can you also add a comment explaining why it's done 😅?

Reply:
Bump

Reply:
Sure 😅 

Done in [7bbb316](https://github.com/software-mansion/react-native-gesture-handler/pull/3154/commits/7bbb31637572b01285849e82a87431fdd36c0728)